### ISSUE TITLE
2.2. wrong instructions for security from Rpi 

### ISSUE THREAD

**Issue by sineverba** [mysensors/MySensors#1055]
I use the software security and signing.

Launching `sudo mysgw --set-soft-hmac-key=CA425435D5765F2A96B97A10B38` Rpi gives following instructions:

```
The next line is intended to be used in SecurityPersonalizer.ino:
#define MY_SOFT_HMAC_KEY	0XCA,0X42,0X54,0X35,0X5F,0X2A,0XB,0X38
```

In SecurityPersonalizer.ino there is no MY_SOFT_HMAC_KEY but `#define MY_HMAC_KEY`

Seems need to edit PI/Linux instructions.

referrer also with this topic: https://forum.mysensors.org/topic/8968/wrong-instructions-on-gateway-for-personalizer

Have a nice day!

**Comment by pepeEL** [comment]
Hi
Please help me. Is any place where i can found all info step by step how secure my gateway on RPi3 and nodes on Arduino MIni Pro with radio RFM69HW ? Please

**Comment by fallberg** [comment]
Fixed in #1056

**Comment by fallberg** [comment]
@pepeEL It is currently not implemented. PR #1056 implements support for activating encryption on RFM69HW but it is untested. Please test.
